assets/js/components/Accordion/Accordion.stories.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import PremiumPill from '@components/PremiumPill';
+import Accordion from '.';
+
+export default {
+  title: 'Accordion',
+  component: Accordion,
```

Comment:
Hey the story works fine as it is, but what do you think about [using storybook argTypes](https://storybook.js.org/docs/react/api/argtypes) ?
We could have something like this which can be passed as args to a Template
```
argTypes: {
    header: {
      type: 'string',
      defaultValue: 'Accordion Header',
      description: 'The title of the accordion component',
      control: {
        type: 'text',
      },
    }
```
```
 export function Default(args) {
  return (
    <Accordion {...args}>
      <div className="p-6">Accordion content</div>
    </Accordion>
  );
}
```   

This would have the nice effect that we can change the text inside storybook, playing with it without touching  the component :D
The rest of the parameters could be added too.

Reply:
sure, will do!

Reply:
@EMaksy I changed as per the documentation of the last storybook's version